### PR TITLE
docs: update docker image

### LINKED ISSUE #766: Download demo docker from dockerhub

### PR DESCRIPTION
**Pull request by dl239**
*  don't upload image to github
*  update doc

### CODE REVIEW

File: .github/workflows/openmldb-docker.yml [hunk 1]
```diff
@@ -80,6 +76,5 @@ jobs:
           push: ${{ env.PUSH }}
           platforms: linux/amd64
           tags: |
-            ${{ env.GHCR_TAG }}
```
Comment (aceforeverd) [inline .github/workflows/openmldb-docker.yml:83]: why rm ghcr ?

Comment (lumianph) [reply .github/workflows/openmldb-docker.yml:83]: @aceforeverd we have decided to make dockerhub as the only place to download docker images, except that complication image required by the cicd (considering the download speed)

**Review by lumianph — APPROVED** [review]

File: .github/workflows/openmldb-docker.yml [hunk 2]
```diff
@@ -45,13 +45,9 @@ jobs:
             echo "PUSH=true" >> $GITHUB_ENV
             IMAGE_ID=ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME
```
Comment (zhanghaohit) [inline .github/workflows/openmldb-docker.yml]: IMAGE_ID can be removed.

Comment (zhanghaohit) [reply .github/workflows/openmldb-docker.yml]: Login to Docker Hub task can also be removed

**Review by zhanghaohit — CHANGES_REQUESTED** [review]

**Review by zhanghaohit — APPROVED** [review]

**Pull request merged**